src/holofuel/__integration_tests__/holo/CreateRequest.holoIntegration.test.js

```diff
@@ -0,0 +1,166 @@
+
+      // wait for home page to reload
+      // TODO: Remove reload page trigger and timeout once resolve signIn/refresh after signUp bug..
+      await page.reload({ waitUntil: ['networkidle0', 'domcontentloaded'] })
+
+      await wait(3000)
+      const buttons = await page.$$('button')
+      const newTransactionButton = buttons[3]
+
+      await waitLoad(() => wsConnected)
+      await waitLoad(() => completeFirstGet)
+
+      // wait for DHT consistency
+      await awaitSimpleConsistency(scenario, DNA_INSTANCE, [counterpartyAgentInstance], [hostedAgentInstance])
+
+      // *********
+      // Create New Request
+      // *********
+      newTransactionButton.click()
+      await page.waitForSelector('button.AmountInput-module__numpad-button___2L0x3')
+      const numpadButtons = await page.$$('button.AmountInput-module__numpad-button___2L0x3')
+      numpadButtons[0].click()
+      await wait(100)
+      numpadButtons[1].click()
+      await wait(100)
+      numpadButtons[2].click()
+      await wait(100)
+      numpadButtons[9].click()
+      await wait(100)
+      numpadButtons[3].click()
+      await wait(100)
+      numpadButtons[4].click()
+      await wait(100)
+      numpadButtons[5].click()
+      await wait(100)
+
+      await wait(1000)
+
+      const actionButtons = await page.$$('button.AmountInput-module__action-button___HfE-d.UIButton-module__button___2eLXd.UIButton-module__white___3J1jF')
+      const requestButton = actionButtons[1]
+      requestButton.click()
+
+      await wait(1000)
+      await page.waitForSelector('form')
+
+      const conterpartyAgentAddress = counterpartyAgentInstance.info(DNA_INSTANCE).agentAddress
+      const newOffer = {
+        amount: '123.456',
+        counterpartyId: conterpartyAgentAddress,
+        note: 'Taco Tuesday!'
+      }
+
+      await page.type(`#counterpartyId`, newOffer.counterpartyId, { delay: 10 })
+      await page.type(`#notes`, newOffer.note, { delay: 100 })
+
+      const amount = await page.$eval(`div.CreateOfferRequest-module__amount___YdE2C`, el => el.innerHTML)
+      const counterpartyId = await page.$eval(`#counterpartyId`, el => el.value)
+      const note = await page.$eval(`#notes`, el => el.value)
+
+      expect(amount).toContain(newOffer.amount)
+      expect(counterpartyId).toBe(newOffer.counterpartyId)
+      expect(note).toEqual(newOffer.note)
+
+      // wait for button to not be disabled
+      await wait(1000)
+
+      const submitButton = await page.$("button[data-testid='submit-button']")
+      submitButton.click()
+
+      // // wait for DHT consistency
+      await awaitSimpleConsistency(scenario, DNA_INSTANCE, [counterpartyAgentInstance], [hostedAgentInstance])
+
+      const checkListPending = async () => counterpartyAgentInstance.call('holofuel', 'transactions', 'list_pending', {})
+      const listPending = await waitZomeResult(checkListPending, 90000, 10000)
+
+      expect(listPending.requests.length).toEqual(1)
```

Comment:
I think we should add a zome call check before the request that calls checkListPending and ensures that the length is 0.